### PR TITLE
Add console outputs and fix some mistakes

### PR DESCRIPTION
**Pull request by jn-lp**

### CODE REVIEW

File: JavaScript/4-for-in-obj.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ hash.fifth = 2;
 for (const key in hash) {
   const value = hash[key];
   console.log(
-    key, '\t', typeof(key), '\t',
-    value, '\t', typeof(value)
+    key, '\t', typeof(key), '\t', // first string ... fifth string //etc.
+    value, '\t', typeof(value) // 7 number ... 2 number //etc.
```
Comment (tshemsedinov) [inline JavaScript/4-for-in-obj.js]: Strange output comments

File: JavaScript/5-for-in-array.js [hunk 2]
```diff
@@ -8,5 +8,5 @@ numbers[5] = 20;
 
 for (const i in numbers) {
   const value = numbers[i];
-  console.log(i, typeof(i), value);
+  console.log(i, typeof(i), value); // 0 string 7 ... field1 string Value1 //etc.
```
Comment (tshemsedinov) [inline JavaScript/5-for-in-array.js]: Gives nothing

File: JavaScript/6-for-of.js [hunk 3]
```diff
@@ -10,3 +10,11 @@ for (const i of numbers) {
...
   console.log(i, typeof(i), value);
 }
+/*
+  7 'number' undefined
+  10 'number' undefined
+  1 'number' 10
+  5 'number' 20
+  2 'number' 1
+  20 'number' undefined
+*/
```
Comment (tshemsedinov) [inline JavaScript/6-for-of.js]: This looks better

File: JavaScript/9-forEach.js [hunk 4]
```diff
@@ -2,15 +2,15 @@
 
 const numbers = [7, 10, 1, 5, 2];
 numbers.forEach((item, i, arr) => {
-  console.log(i, arr, item);
+  console.log(i, arr, item); // 0 [7, 10, 1, 5, 2] 7 ... 4 [7, 10, 1, 5, 2] 2 //etc.
```
Comment (tshemsedinov) [inline JavaScript/9-forEach.js]: Bad style, long lines, not informative

File: JavaScript/a-map.js [hunk 5]
```diff
@@ -16,12 +14,11 @@ const res1 = [7, 10, 1, 5, 2]
...
-  .map(x => x * 2)
-  .map(log)
-  .map(x => ++x)
-  .map(log);
+  [7, 10, 1, 5, 2]
+    .map(log) // 0 7 // 1 10 // 2 1 // 3 5 // 4 2
+    .map(x => x * 2)
+    .map(log) // 0 14 // 1 20 // 2 2 // 3 10 // 4 4
+    .map(x => ++x)
+    .map(log); // 0 15 // 1 21 // 2 3 // 3 11 // 4 5
```
Comment (tshemsedinov) [inline JavaScript/a-map.js]: wtf?

**Comment by jn-lp** [comment]
The most of the changes were written on the go in standard code editors, not using a terminal - just the brain. -This is why I used `CRLF `and no `Eslint`. As for the style of comments - understood, accepted.

**Comment by tshemsedinov** [comment]
@jn-lp see https://help.github.com/articles/dealing-with-line-endings/ and github has builtin simple editor for such situations working in web.

**Comment by jn-lp** [comment]
@tshemsedinov, ok thanks

**Pull request closed without merging**